stdlib/websub/src/main/ballerina/src/websub/commons.bal

```diff
@@ -477,9 +477,7 @@ public type RemotePublishConfig record {|
 #               The publish URL is defined as {publicUrl}/{basePath}/{publishResourcePath} if `publicUrl` is
 #               specified, defaults to `http(s)://localhost:{port}/{basePath}/{publishResourcePath}` if not.
 # + hubConfiguration - The hub specific configuration
-# + return - `Hub` The WebSub Hub object representing the newly started up hub, or `HubStartedUpError` indicating
-#            that the hub is already started, and including the `websub:Hub` object representing the
-#            already started up hub
+# + return - A newly started WebSub Hub or else a `websub:HubStartedUpError` if the hub is already started
```

Comment:
Again you have removed very important information by trying to adhere to the guidelines? Please undo this. Check other places as well